### PR TITLE
Fix #21 and add option to show your plays

### LINKED ISSUE #21: Users with no avatar don't display correctly
Example: image

### PR DESCRIPTION
**Pull request by Hum4n01d**
- Fixes placeholder url for Last.fm friends with no avatar
- Adds option to show your plays
image
When checked:
image
(featuring fixed placeholder image!)

### CODE REVIEW

File: src/components/Content.jsx [hunk 1]
```diff
@@ -7,6 +7,7 @@ import * as lastfm from './../lastfm';
 import * as storage from './../storage'
 import determineType from './../determineType';
 import { each, eachLimit } from 'async'
+import * as cache from '../storage';
```
Comment (xxdavid) [inline src/components/Content.jsx]: `storage.js` is already imported on line 7 as `storage` so I think this line isn't needed. Am I wrong?

File: src/components/Content.jsx [hunk 2]
```diff
@@ -35,6 +36,13 @@ export default class Content extends React.Component {
         var username = parser.getUsername();
 
         lastfm.fetchFriends(username, (friends) => {
+            if (cache.getShowYourPlays()) {
```
Comment (xxdavid) [inline src/components/Content.jsx]: Please change the variable `cache` to `storage` if there's no reason for the duplicate import.

**Review by xxdavid — COMMENTED** [review]
Hi, thank you very much for the pull request! The code looks good, I have just two minor comments about duplicate import of `storage.js`. Could you please fix that and update the PR (by pushing new commits to your branch)? I'll merge it afterwards. :)

Side note: it would be maybe better to separate the changes into two PRs (for #21 and #22) but it's no problem here.

**Comment by Hum4n01d** [comment]
Whoops! Good catch, fixed 👍

**Comment by xxdavid** [comment]
Thank you, I'm gonna merge it!

**Comment by xxdavid** [comment]
I released a new version (1.3) and submitted it to both stores. 🎉

**Pull request merged**